wallet/wallet.go

```diff
@@ -106,6 +109,23 @@ func (w *Wallet) SignBytes(data []byte, addr address.Address) (types.Signature,
 	return backend.SignBytes(data, addr)
 }
 
+// AddressFromPubKey looks up a KeyInfo address associated with a given PublicKey
+func (w *Wallet) AddressFromPubKey(pk []byte) (address.Address, error) {
```

Comment:
nit: naming should be consistent across GetPubKeyForAddress and AddressFromPubKey.  Both should use or not use get, both should use from or for

Reply:
good catch, thanks